### ISSUE TITLE
Paginated list jumps to first page on data change

### ISSUE THREAD

**Issue by eybarta** [TahaSh/vue-paginate#58]
Hi, thx for your great plugin.

1 issue I have, let's say I'm on page 10 and I changed data to one of
the items.. the pager jumps back to page 1.. Is this the default behaviour?

Is there a way to change this behaviour?

thx!

**Comment by lesenther** [comment]
+1

Currently doing this as a work around:

Add ref attribute in paginate component.  E.g.:
```
<paginate ... ref="paginatedData">
```

Add in methods:
```
changeItem () {
  let currentPage = this.$refs.paginatedData.currentPage + 1
  let _this = this

  // change item

  setTimeout(function () {
    _this.$refs.paginatedData.goToPage(currentPage)
  }, 1)
}
```

**Comment by apegadoboureghida** [comment]
I created a pull request yesterday with a new feature to support this. I don't know if @TahaSh is still maintaining this repo but you can try this feature with 

> npm install https://github.com/apegadoboureghida/vue-paginate.git --save

and to use it:
`<paginate name="items" :list="items" :refreshCurrentPage="false">`

**Comment by ilterocal** [comment]
I have the same problem, i don't want to navigate to the first page when my list size gets changed.

**Comment by gemp** [comment]
Many many thanks @apegadoboureghida

**Comment by TahaSh** [comment]
This issue is officially fixed in version `3.6.0`.
